source/Tests/Playnite.Tests/Scripting/PowerShell/PowerShellTests.cs

```diff
@@ -68,18 +68,23 @@ public void ErrorHandlingTest()
         }
 
         [Test]
-        public void GetFunctionExitsTest()
+        public void GetFunctionTest()
         {
-            using (var ps = new PowerShellRuntime("GetFunctionExitsTest"))
+            using (var tempDir = TempDirectory.Create())
             {
-                Assert.IsTrue(ps.GetFunction("TestFunc") == null);
-                ps.Execute(@"
+                using (var ps = new PowerShellRuntime("GetFunctionTest"))
+                {
+                    Assert.IsTrue(ps.GetFunction("TestFunc") == null);
+                    var path = Path.Combine(tempDir.TempPath, "GetFunctionTest.psm1");
+                    File.WriteAllText(path, @"
 function TestFunc()
 {
     return 4 + 4
 }
 ");
-                Assert.IsTrue(ps.GetFunction("TestFunc") != null);
+                    ps.ImportModule(path);

```

Comment:
I modified this test to create a `.psm1` file and import it as a module. Exported functions will _only_ be found if a module import is done. If you only eval a script and define functions the exported functions will not be detected by `PowerShellRuntime`.